public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -94,16 +104,23 @@ export class LogRowMessage extends PureComponent<Props> {
             className={cx('log-row-menu', styles.rowMenu, {
               [styles.rowMenuWithContextButton]: shouldShowContextToggle,
             })}
-            onClick={(e) => e.stopPropagation()}
+            onClick={this.onLogRowClick}
           >
             {shouldShowContextToggle && (
               <Tooltip placement="top" content={'Show context'}>
                 <IconButton size="md" name="gf-show-context" onClick={this.onShowContextClick} />
               </Tooltip>
             )}
-            <Tooltip placement="top" content={'Copy'}>
-              <IconButton size="md" name="copy" onClick={() => navigator.clipboard.writeText(restructuredEntry)} />
-            </Tooltip>
+            <ClipboardButton
+              className={styles.copyLogButton}
+              icon="copy"
+              variant="secondary"
+              fill="text"
+              size="sm"
```

Comment:
```suggestion
              size="md"
```

To keep the size of `showContext` and `Copy` the same